warning: Git tree '/project/builds/hammer_build_uxsim_1.5.0/build' is dirty
trace: dill matched to 0.3.1.1
trace: numpy matched to 2.1.0
trace: pandas matched to 2.2.2
trace: pillow matched to 10.4.0
trace: pyqt5 matched to 5.15.11
trace: python-dateutil matched to 2.9.0.post0
trace: pytz matched to 2024.1
trace: tzdata matched to 2024.1
trace: pyqt5-qt5 matched to 5.15.15
trace: six matched to 1.16.0
these 3 derivations will be built:
  /nix/store/5ljn80n0d2f2y4bfq1i81i665zlc7k83-uv2nix-hammer-app-0.1.0.drv
  /nix/store/hw3irap51ml5pry04nbmclvmnqklakd3-pyqt5-5.15.11.drv
  /nix/store/9war3a1cvhriy9q354bj6s26hpv89p3z-test-venv.drv
building '/nix/store/hw3irap51ml5pry04nbmclvmnqklakd3-pyqt5-5.15.11.drv'...
building '/nix/store/5ljn80n0d2f2y4bfq1i81i665zlc7k83-uv2nix-hammer-app-0.1.0.drv'...
error: builder for '/nix/store/hw3irap51ml5pry04nbmclvmnqklakd3-pyqt5-5.15.11.drv' failed with exit code 1;
       last 25 log lines:
       >     raise BackendUnavailable(data.get('traceback', ''))
       > pyproject_hooks._impl.BackendUnavailable: Traceback (most recent call last):
       >   File "/nix/store/rbyrg3i7s7d6z3k94c1x1s6gs1cl83iq-pyproject-hooks-1.0.0/lib/python3.12/site-packages/pyproject_hooks/_in_process/_in_process.py", line 77, in _build_backend
       >     obj = import_module(mod_path)
       >           ^^^^^^^^^^^^^^^^^^^^^^^
       >   File "/nix/store/h3i0acpmr8mrjx07519xxmidv8mpax4y-python3-3.12.5/lib/python3.12/importlib/__init__.py", line 90, in import_module
       >     return _bootstrap._gcd_import(name[level:], package, level)
       >            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
       >   File "<frozen importlib._bootstrap>", line 1387, in _gcd_import
       >   File "<frozen importlib._bootstrap>", line 1360, in _find_and_load
       >   File "<frozen importlib._bootstrap>", line 1310, in _find_and_load_unlocked
       >   File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
       >   File "<frozen importlib._bootstrap>", line 1387, in _gcd_import
       >   File "<frozen importlib._bootstrap>", line 1360, in _find_and_load
       >   File "<frozen importlib._bootstrap>", line 1331, in _find_and_load_unlocked
       >   File "<frozen importlib._bootstrap>", line 935, in _load_unlocked
       >   File "<frozen importlib._bootstrap_external>", line 995, in exec_module
       >   File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
       >   File "/nix/store/j8n0748n4f96v18sqw0lh222piw90nkh-sip-6.8.6/lib/python3.12/site-packages/sipbuild/__init__.py", line 19, in <module>
       >     from .setuptools_builder import SetuptoolsBuilder
       >   File "/nix/store/j8n0748n4f96v18sqw0lh222piw90nkh-sip-6.8.6/lib/python3.12/site-packages/sipbuild/setuptools_builder.py", line 8, in <module>
       >     from setuptools import Extension, setup
       > ModuleNotFoundError: No module named 'setuptools'
       >
       > ERROR Backend 'sipbuild.api' is not available.
       For full logs, run 'nix log /nix/store/hw3irap51ml5pry04nbmclvmnqklakd3-pyqt5-5.15.11.drv'.
error: 1 dependencies of derivation '/nix/store/9war3a1cvhriy9q354bj6s26hpv89p3z-test-venv.drv' failed to build
